app/stores/ada/TrezorConnectStore.js

```diff
@@ -10,10 +10,9 @@ import environment from '../../environment';
 
 import TrezorConnect, { UI_EVENT, DEVICE_EVENT } from 'trezor-connect';
```

Comment:
try to pre-initialize using `TrezorConnect.init()`

Benefits:
1. User will have to wail bit lesser for `TrezorConnect API` to respond
2. User can close `TrezorConnect` tab before it's fully initialize, as a result:
   A. sometimes `TrezorConnect API` hangs because of this, so it will be solved
   B. UX will be improved

Reply:
DONE